### ISSUE TITLE
Unable to select from a long drop-down list in a touch device (iPad & Android tablets)

### ISSUE THREAD

**Issue by mbeneto** [google/blockly#289]
Dear all.

I have experienced a problem when trying to select an item from a long drop-down list in a touch device. If the drop-down list is longer than the initial scroll, the user cannot select properly items outside this position. I have tried both in Android tablets and iPad, using for tests Chrome and Safari, having this behaviour in all configurations.

This problem doesn't appear when displaying Blockly from a non-touch device, and the user can scroll properly the drop-down list and select the desired item without any problem.

I attach you an example with a custom block with a 60 items drop-down list. I suggest you to try to select one of the last list items and you will visualize immediately the problem.

[test_dropdown.zip](https://github.com/google/blockly/files/177395/test_dropdown.zip)

Thanks. Best regards,
Marc.

**Comment by mbeneto** [comment]
Hi again.

I come back to you with another example where the problem is easier detected.

I added a regular HTML select element of 60 items. It is possible to scroll the whole list and select the desired element without any problem from tactile devices, however, as I pointed in my first message, the same drop-down list inside a Blockly element is not possible to scroll it and select the desired item properly. Any idea or workaround to solve this problem?

[test_dropdown_2.zip](https://github.com/google/blockly/files/191295/test_dropdown_2.zip)

Thanks.
Marc.
